docs/shared/ioda.yaml

```
@@ -0,0 +1,25 @@
+ioda:
+  configuration_file:
+    base_file: path/to/config.yaml
```

```suggestion
    base_file: /path/to/config.yaml
```
This could also be updated, for consistency with other files, in `jedi.yaml`.